spec/datadog/appsec/configuration/settings_spec.rb

```diff
@@ -77,6 +77,30 @@ def patcher
       end
     end
 
+    describe '#rasp_enabled' do
+      subject(:rasp_enabled) { settings.appsec.rasp_enabled }
+
+      context 'when DD_APPSEC_RASP_ENABLED' do
```

Comment:
I think this sentence is missing the verb ... if I reed it it's like `when specific setting` ... but specific setting what?